pkg/decision/evaluator/condition.go

```diff
@@ -45,9 +45,15 @@ func NewCustomAttributeConditionEvaluator(logger logging.OptimizelyLogProducer)
 // Evaluate returns true if the given user's attributes match the condition
 func (c CustomAttributeConditionEvaluator) Evaluate(condition entities.Condition, condTreeParams *entities.TreeParameters, options *decide.Options) (bool, decide.DecisionReasons, error) {
 	// We should only be evaluating custom attributes
-
 	reasons := decide.NewDecisionReasons(options)
-	if condition.Type != customAttributeType {
+	isValid := false
+	for _, validType := range validTypes {
```

Comment:
why nchanged?

Reply:
we have two valid types that we support now, this can be seen here: https://github.com/optimizely/go-sdk/pull/350/files#diff-4b69e47626e15aedb2e8d1bb3b34fdb04fd57af036e7c00f9f968a5490d24750R35